src/Microsoft.Diagnostics.Monitoring.WebApi/StreamingLogger.cs

```diff
@@ -67,27 +67,38 @@ public IDisposable BeginScope<TState>(TState state)
 
         public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Exception exception, Func<TState, Exception, string> formatter)
         {
-            if (_logFormat == LogFormat.Json)
+            if (_logFormat == LogFormat.NDJson)
             {
                 LogJson(logLevel, eventId, state, exception, formatter);
             }
+            else if (_logFormat == LogFormat.JsonSequence)
+            {
+                LogJson(logLevel, eventId, state, exception, formatter, LogFormat.JsonSequence);
+            }
             else
             {
                 LogEventStream(logLevel, eventId, state, exception, formatter);
             }
         }
 
-        private void LogJson<TState>(LogLevel logLevel, EventId eventId, TState state, Exception exception, Func<TState, Exception, string> formatter)
+        private void LogJson<TState>(LogLevel logLevel, EventId eventId, TState state, Exception exception, Func<TState, Exception, string> formatter, LogFormat jsonFormat = LogFormat.NDJson)
         {
             Stream outputStream = _outputStream;
 
+            
+            if (jsonFormat == LogFormat.JsonSequence)
+            {
+                const byte recordSeparator = 0x1E;
```

Comment:
nit: Let's make this a constant on the class and rename to something like `JsonSequenceRecordSeparator`.